javascript/ql/test/query-tests/Security/CWE-730/RegExpInjection.js

```diff
@@ -1,6 +1,6 @@
 var express = require('express');
 var app = express();
-
+var URI = reuires("urijs");
```

Comment:
@esben-semmle This looks like a typo from https://github.com/Semmle/ql/pull/158. If so, worth fixing on master and including in the next mergeback.